paper-typeahead-input.html

```diff
@@ -313,7 +340,7 @@
         }
       });
       var input = Polymer.dom(this.root).querySelector('paper-input');
-      input.$.input.autocomplete = 'address-level4';
+      input.$.input.autocomplete = this.autocomplete;
```

Comment:
Ок, no problem.But this solution is working for me.

On 18 July 2016 at 17:13, Tianxiang Zhang <notifications@github.com> wrote:

> In paper-typeahead-input.html
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30#discussion_r71153857>
> :
>
> > @@ -313,7 +340,7 @@
> >          }
> >        });
> >        var input = Polymer.dom(this.root).querySelector('paper-input');
> > -      input.$.input.autocomplete = 'address-level4';
> > +      input.$.input.autocomplete = this.autocomplete;
>
> To prevent autofilling of Chrome, autocomplete="Off" is not a reliable
> solution. Chrome Browser Ignoring AutoComplete=Off. See discussion here
> http://stackoverflow.com/questions/12374442/chrome-browser-ignoring-autocomplete-off.
> And plus, the expected behavior of paper-typeahead-input is to provide its
> own suggest, so I don't see why we need to expose this autocomplete
> property of inner input as the property of paper-typeahead-input.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30/files/dfdc34b0472906d1f9c7005d4442b0162ab8c4f8#r71153857>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG0SnBPJdniRjFkkSU_AG-yL2I-8STNQks5qW4n-gaJpZM4JKWUL>
> .
>



-- 
C уважением, Пличко Павел